docs/persistent-volumes.md

```diff
@@ -0,0 +1,170 @@
+# Persistent Volumes and Claims
+
+This document describes the current state of Persistent Volumes in Kubernetes.  Familiarity with [volumes](./volumes.md) is suggested.
+
+A Persistent Volume (PV) is a piece of networked storage in the cluster that has been provisioned by an administrator.  It is a resource in the cluster just like a node is a cluster resource.   PVs are volume plugins like Volumes, but have a lifecycle independent of any individual pod that uses the PV.
+
+A Persistent Volume Claim (PVC) is a request for storage by a user.  It is similar to a pod.  Pods consume node resources and PVCs consume PV resources.  Pods can request specific levels of resources (CPU and Memory).  Claims can request specific size and access modes (e.g, can be mounted once read/write or many times read-only).  
+
+Please see the [detailed walkthrough with working examples](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/persistent-volumes).
+
+
+## Lifecycle of a volume and claim
+
+PVs are resources in the cluster.  PVC are requests for those resources and also act as claim checks to the resource.  The interaction between PVs and PVCs follows this lifecycle:
+
+### Provisioning
+	
+The volume is created by an administrator.  It becomes a cluster resource available for consumption.
+
+### Binding
+
+A persistent volume claim is created by a user requesting a specific amount of storage and with certain access modes.  There is a process watching for new claims that binds them to an available volume if a match is available.  The user will always get at least what they asked for, but the volume may be in excess of what was requested.
```

Comment:
Use active voice when you can.  "A user creates a PVC" rather than "A PVC is created".

"There is a process watching" -> "A control loop in the master watches for new PVCs, finds an appropraite PV (if possible) and binds them together."